alpha/hal.cpp

```diff
@@ -31,8 +33,9 @@ void _TS_HAL::begin()
   this->random_seed();
 
 #ifdef HAL_M5STICK_C
+  
   ENTER_CRITICAL;
-  M5.begin();
+  M5.begin(true, true, false);
```

Comment:
@fustilio @yupeng-zglue do we know why?

Reply:
By default it's all true, setting to false prevents the setup of serial in the begin function. This is so that the serial is setup later in the code.